src/utils/sendMessages.js

```diff
@@ -14,18 +14,16 @@ const sleep = promisify(setTimeout);
  * @param {Array} blocks Slack "blocks" to send via message
  */
 const sendDelayedMessages = async function sendMessages(sender, blocks) {
-  await Promise.all(
-    blocks.map(async (blockChunk, i) => {
-      // The slack rate limit for messages is 1 per second, unless it is defined in
-      // "special rate limits".
-      if (i !== 0) await sleep(1500);
-      try {
-        sender(blockChunk);
-      } catch (error) {
-        console.log(error);
-      }
-    }),
-  );
+  for (const [i, blockChunk] of blocks.entries()) {
+    // The slack rate limit for messages is 1 per second, unless it is defined in
+    // "special rate limits".
+    if (i !== 0) await sleep(1500);
+    try {
+      await sender(blockChunk);
```

Comment:
Do you think this `await` is necessary in practice with the delay above? I wonder if it would be faster but still stable without. We could use `.catch()` to handle the error.